### PR TITLE
feature/logo

### PR DESCRIPTION
**Pull request by Teque5**
@bhilburn 

## Features

- Adds `sigmf` logo & png
- Removes `example_metadata.py`

This PR includes the SigMF logo as discussed in #117 but should be modified to replace "(@bhilbern insert)" in the provided `logo/README.md` with the path to the final zip archive.

This PR only has one commit but branches from PR #138, so that should be merged first.

## Option
One option I'll mention - I was considering making it so the logo would be installed with the `pip` package (when future users do `pip install sigmf`. This is easily possible but means that the `sigmf` files must live in a subfolder of the package to be installed. I wasn't sure if we wanted to do this. We could potentially use it for tests or examples out-of-the-box if we go that direction. It would mean our distributable would go from 19KB to 1.2MB.  `¯\_(ツ)_/¯ `

*Note to self* to implement the `pip` distributed logo:

- move the `sigmf` files to a folder inside the module
- add `'logo': ['*.sigmf*'],` to `package_data` inside `setup.py`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,18 +44,31 @@ discussions, the more useful the standard is likely to be.
...
 ## Use Cases
+
+### Load a SigMF archive; read all samples & metadata
+
+```python
+import sigmf
+handle = sigmf.sigmffile.fromfile('example.sigmf')
+handle.read_samples() # returns timeseries data
+handle.get_global_info() # returns 'global' metadata dictionary
+handle.get_captures() # returns 'captures' metadata dictionary
```
Comment (jacobagilbert) [inline README.md:61]: Minor note: get_captures:

```
# returns list of 'captures' dictionaries
```

**Comment by gmabey** [comment]
Sure looking forward to this getting merged!

**Comment by Teque5** [comment]
Fixed @jacobagilbert's minor issue by updating the other PR #138 and rebasing this.

**Comment by bhilburn** [comment]
@gmabey - Yes! But it depends on #138, which must get merged first, and there are a couple of final things on that branch to clean-up prior to merge 🙂

**Comment by bhilburn** [comment]
@Teque5 - Actually, I just realized the comment I left in #117 is wrong - I should /not/ touch the README, because it'll just create new conflicts with this PR, hah. 

Do you want to update this PR with the final path? It's here:
`https://github.com/gnuradio/SigMF/wiki/logo/sigmf_logo_files.zip`

**Comment by Teque5** [comment]
Okay this branch is now ready to merge after #138. I updated the link in the README.md.

**Comment by Teque5** [comment]
I think you may have to grant me edit access on the wiki.

**Comment by bhilburn** [comment]
Merged. @Teque5 - just added you to a team to try and give you wiki write access. Give it a go

**Pull request merged**